src/table.js

```diff
@@ -578,13 +578,54 @@ Table.prototype.delete = function(gaxOptions, callback) {
   );
 };
 
+/**
+ * Truncate the table.
+ *
+ * @param {object} [gaxOptions] Request configuration options, outlined
+ *     here: https://googleapis.github.io/gax-nodejs/CallSettings.html.
+ * @param {function} callback The callback function.
+ * @param {?error} callback.err An error returned while making this request.
+ * @param {object} callback.apiResponse The full API response.
+ *
+ * @example
+ * table.truncate(function(err, apiResponse) {});
+ *
+ * //-
+ * // If the callback is omitted, we'll return a Promise.
+ * //-
+ * table.truncate().then(function(data) {
+ *   var apiResponse = data[0];
+ * });
+ */
+Table.prototype.truncate = function(gaxOptions, callback) {
+  if (is.fn(gaxOptions)) {
+    callback = gaxOptions;
+    gaxOptions = {};
+  }
+
+  var reqOpts = {
+    name: this.id,
+    deleteAllDataFromTable: true,
+  };
+
+  this.bigtable.request(
+    {
+      client: 'BigtableTableAdminClient',
+      method: 'dropRowRange',
+      reqOpts: reqOpts,
+      gaxOpts: gaxOptions,
+    },
+    callback
+  );
+};
+
 /**
  * Delete all rows in the table, optionally corresponding to a particular
  * prefix.
  *
- * @param {object} [options] Configuration object.
- * @param {string} [options.prefix] Row key prefix, when omitted all rows
- *     will be deleted.
+ * @param {string} [prefix] Row key prefix. This field is required.
```

Comment:
This can just be `prefix` now. The surrounding brackets indicate that it's optional.

Reply:
(you can also remove `This field is required.` as that will be implied.

Reply:
done